.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,73 @@
+<!--
+PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.
+
+ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
+-->
+
+## I'm submitting a...
```

Comment:
I'd rather use Github's ability to have multiple templates for different kinds of Github issues. That way we don't ask people to answer questions that don't apply to what they're doing (such as "Expected behavior" when they're just wanting to ask a question).